### ISSUE TITLE
Changing anything on electron.exe breaks camera

### ISSUE THREAD

**Issue by promosjotte** [electron/rcedit#73]
The app uses navigator.mediaDevices.getUserMedia to the camera. The App is copied to the prebuilt files and working, as long as I change anything on the electron.exe file.
After that the internal camera can not be used anymore.
Any possible solution? Or at least an explanation?

**Comment by apopovich** [comment]
Same issue - after rcedit modified electron.exe got navigator.mediaDevices.getUserMedia error "Could not start audio source"

**Comment by altis10** [comment]
Can you check if the application's size is still the one before rcedit changed it?
I got similar error and, when checked, the rcedit app was somehow modifying the size of the exe file....

**Comment by apopovich** [comment]
> Can you check if the application's size is still the one before rcedit changed it?

No, size is different. Even if run rcedit without args and not modify resources.

> I got similar error and, when checked, the rcedit app was somehow modifying the size of the exe file....

Did you find solution?

**Comment by altis10** [comment]
Unfortunately not yet...
